actions/websocket_actions.jsx

```diff
@@ -164,6 +165,9 @@ function reconnectWebSocket() {
     initialize();
 }
 
+window.closeWS = close;
+window.connectWS = reconnect;
```

Comment:
why do we need to make this available? I don't see it being used in the rest of the PR

Reply:
Sorry, bad debug commit. I'll remove.